tests/system/auth.go

```diff
@@ -29,11 +29,40 @@ var (
 	token = "INVALID_TOKEN"
 )
 
+func addUser(u sdktypes.User) string {
+	return fmt.Sprintf(
+		`insert into users(user_id,email,display_name,created_by,default_org_id) values (%q,%q,%q,%q,%q)`,
```

Comment:
Never ever EVER use `fmt.Sprintf` to construct a DB query. Not even in test code like here. The potential for abuse is too much - even in test code.

The CodeQL failure below is not a false positive. Either use Gorm, or (less desirable, but acceptable in test code) the suggested fix below.

Reply:
now using objects to populate the db